functions/java-jre.sh

```diff
@@ -17,7 +17,28 @@ java_webupd8() {
   if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; exit 1; fi
 }
 
-java_zulu_embedded() {
+java_zulu() {
+  FILE="$(mktemp /tmp/.XXXXXXXXXX)"
+
+  if is_arm; then
+    JAVA=zulu8.31.1.122-jdk1.8.0_181-linux_aarch32hf
+  else
+    JAVA=zulu8.31.0.1-jdk8.0.181-linux_x64
```

Comment:
Should we assume here that anything not arm is 64-bit?

Reply:
We have no check for 32/64 bit and latest Zulu is not available in 64 bit on x86 so I think it is somewhat safe to assume that we _want_ this to be 64bit always, wdyt?